otx/algorithms/common/adapters/mmdeploy/ops/custom_ops.py

```diff
@@ -0,0 +1,80 @@
+"""Custom patch of mmdeploy ops for openvino export."""
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import torch
+from mmdeploy.core import SYMBOLIC_REWRITER
+from mmdeploy.utils import get_ir_config
+from torch.nn.functional import (
+    GRID_SAMPLE_INTERPOLATION_MODES,
+    GRID_SAMPLE_PADDING_MODES,
+)
+from torch.onnx import symbolic_helper
+from torch.onnx._internal import jit_utils
+
+# Remove previous registered symbolic
+SYMBOLIC_REWRITER._registry._rewrite_records["squeeze"] = list()
+SYMBOLIC_REWRITER._registry._rewrite_records["grid_sampler"] = list()
+
+
+@SYMBOLIC_REWRITER.register_symbolic("squeeze", is_pytorch=True)
+def squeeze__default(ctx, g, self, dim=None):
```

Comment:
What does the double underbar mean?

Reply:
I don't know... I think it is convention from mmdeploy.